include/nccl_ofi_msgbuff.h

```diff
@@ -84,11 +86,13 @@ typedef struct {
 } nccl_ofi_msgbuff_t;
 
 /**
- * Allocates and initializes a new message buffer. Buffer size should be a power of 2.
+ * Allocates and initializes a new message buffer.
+ * @param max_inprogress number of elements in storage buffer
+ * @param size number of elements tracked by this msgbuff
  *
  * @return a new msgbuff, or NULL if initialization failed
  */
-nccl_ofi_msgbuff_t *nccl_ofi_msgbuff_init(uint16_t buffer_size);
+nccl_ofi_msgbuff_t *nccl_ofi_msgbuff_init(uint16_t max_inprogress, uint16_t size);
```

Comment:
From reading the documentation for this function, it's not really clear why we need to set both max_inprogress and size.  It's even less clear what these actually mean.  I think the docs need some behavior description.  And probably different names (see above comment).

Reply:
I improved the documentation to make it clearer what this is doing. Let me know if it still isn't clear enough.